evalml/pipelines/time_series_pipeline_base.py

```diff
@@ -67,8 +68,18 @@ def fit(self, X, y):
 
         Returns:
             self
+
+        Raises:
+            ValueError: If the target is not numeric.
```

Comment:
This should be more specific - the target doesn't need to be numeric in all cases, just regression cases, right?

Reply:
Yes specifically just time series regression in this function